### DISCUSSION TITLE
Jagged look of fonts on Windows

### DISCUSSION THREAD

**Discussion by isopterix** [emdgroup-liquid/liquid#275] [Q&A] [answered]
Is there anything we can do to make the fonts inside any of the controls look not as jagged when rendered on a Windows machine? Any recommendation?

**Comment by borisdiakur** [comment, marked as answer]
Hi @isopterix , sorry for being so late with my reply. We'll have a look at this.

**Reply by borisdiakur** [reply]
Tracking this in https://github.com/emdgroup-liquid/liquid/issues/278 now.
